virtcontainers/fc.go

```diff
@@ -143,9 +157,47 @@ func (fc *firecracker) trace(name string) (opentracing.Span, context.Context) {
 	return span, ctx
 }
 
+// bindMount bind mounts a source in to a destination. This will
+// do some bookkeeping:
+// * evaluate all symlinks
+// * ensure the source exists
+// * recursively create the destination
```

Comment:
Can you add a note to specify that you are adding this to override the bind-mount behaviour in Kata to use a slave mount instead?

Reply:
@amshinde you are right. This is only for firecracker specific. Also it does not impact firecracker as it does not use 9p today.